obj_files.mk

```diff
@@ -581,7 +581,7 @@ TEXT_O_FILES +=\
     $(BUILD_DIR)/asm/dolphin/mtx/vec.s.o\
 
 TEXT_O_FILES +=\
-    $(BUILD_DIR)/src/dolphin/os/OS.c.o\
+    $(BUILD_DIR)/src/dolphin/os/OSInit.c.o\
```

Comment:
OS.c is the correct file name as provided by debug maps

Reply:
Well that's fair but we use wine to compile so something has to be done about `os.h` if we want `OS.h`.

Reply:
Hmm that's true. I suppose we could do dolphin/os/OS.h for this TU and dolphin/os.h as the module catchall, but that would require restructuring.

Reply:
Long-term all of `dolphin` is being outsourced, so personally I don't have a problem with naming this wrong temporarily. But I leave the decision up to you, I was just trying to get it to compile.